### ISSUE TITLE
Image detection example Mean AP = 0.0000

### ISSUE THREAD

**Issue by evgenibers** [Microsoft/CNTK#1228]
Hello, I tried Image Detection FastRCNN example and got good results. But when I tried to adapt it for my images I always get Mean AP = 0.0000.

This is output after second epoch:

> Starting Epoch 2: learning rate per sample = 0.000010  effective momentum = 0.900000  momentum as time constant = 9.5 samples
> Starting minibatch loop.
> Finished Epoch[ 2 of 3]: [Training] ce = 10.38870070 * 9; errs = 0.022% * 9; totalSamplesSeen = 18; learningRatePerSample = 9.9999997e-006; epochTime=163.119s
> SGD: Saving checkpoint model './Output/Fast-RCNN.model.2'

And output image:
![o](https://cloud.githubusercontent.com/assets/12796628/21590486/48e24c12-d10d-11e6-926b-9932f2454548.png)

I used version 2.0 beta3 Windows 64bit CPU. Any idea why it so?

**Comment by Vedaevolution** [comment]
The roi generation seems okay. Did you encode your ground truth the right way? 
You can chekc this with the C... scripts included in the package.

Best Regards
Stephen

**Comment by evgenibers** [comment]
Yes, I ran these scripts, and this is the input image with label on it visualized with B1_VisualizeInputROIs.py
![zxc](https://cloud.githubusercontent.com/assets/12796628/21616819/23df634a-d1f4-11e6-8c41-1b6f32196be7.png)
Should be label in the center of the object?

**Comment by Vedaevolution** [comment]
To me it looks like you did encode the ground truth boxes wrong. The red box with label should be wrapped around the object. It seems your ground thruth box has nearly no height and width. With that in mind it's clear tehre is no good learning result.

**Comment by mahilleb-msft** [comment]
@pkranen - what do you think?

**Comment by pkranen** [comment]
I fully agree to Vedaevolution's assessment. The bbox annotation should surround the entire object, see these examples: https://docs.microsoft.com/en-us/cognitive-toolkit/Object-Detection-using-Fast-R-CNN#summary. Closing this issue, feel free to reopen if required.
